fs/dafsserver/dafsserver.cpp

```diff
@@ -5565,7 +5565,11 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
                             ssock.setown(createSecureSocket(sockSSL.getClear(), disableClientCertVerification));
                             int status = ssock->secure_accept();
                             if (status < 0)
-                                throw createDafsException(DAFSERR_serveraccept_failed,"Failure to establish secure connection");
+                            {
+                                if (status != SSL_ERROR_PORT_CHECK)
+                                    status = DAFSERR_serveraccept_failed;
+                                throw createDafsException(status, "Failure to establish secure connection");
```

Comment:
Consider throwing a completely different exception for port check, including different text.  Even though it won't get logged, it would be more "self documenting".

Reply:
ok, done